messaging/rust/Cargo.toml

```diff
@@ -0,0 +1,30 @@
+[package]
+name = "wasmcloud-interface-messaging"
+version = "0.1.1"
+description = "Messaging (wasmcloud:messaging)"
+repository = "https://github.com/wasmcloud/interfaces"
+homepage = "https://github.com/wasmcloud/wasmcloud"
+keywords = ["wasmcloud","capability"]
```

Comment:
Maybe add interface as a keyword to all these crates?

Reply:
sure. we get up to 5.